hdf5.spec

```diff
@@ -406,8 +406,14 @@ do
   mkdir -p ${RPM_BUILD_ROOT}%{_libdir}/hdf5/$mpi/tests/{,.libs/}
   for x in t_cache testphdf5 t_mpi t_pflush1 t_pflush2 t_shapesame
   do
-    install -m 0755 $mpi/testpar/${x} ${RPM_BUILD_ROOT}%{_libdir}/hdf5/$mpi/tests/
-    install -m 0755 $mpi/testpar/.libs/${x} ${RPM_BUILD_ROOT}%{_libdir}/hdf5/$mpi/tests/.libs/
+    if [ "$mpi" = "mpich" ]; then
+      # Maintain backward compatibily for existing HDF5 tests 
+      install -m 0755 $mpi/testpar/${x} ${RPM_BUILD_ROOT}%{_libdir}/hdf5/tests/
+      install -m 0755 $mpi/testpar/.libs/${x} ${RPM_BUILD_ROOT}%{_libdir}/hdf5/tests/.libs/
+    else
```

Comment:
Why is this necessary?  This new packaging is named differently than the old packaging, so existing PRs and master should use the existing/old packages and these new packages, with the new paths won't be used until daos is updated to use the new names.